doc/man3/EC_KEY_new.pod

```diff
@@ -19,6 +19,10 @@ EC_KEY objects
 
  #include <openssl/ec.h>
 
+Deprecated since OpenSSL 3.0, can be hidden entirely by defining
+B<OPENSSL_API_COMPAT> with a suitable version value, see
+L<openssl_user_macros(7)>:
+
```

Comment:
Didn't we at some point decide not to depreciate key *creation* functions yet?
Low level key generation functions and functions that use these keys are up for deprecation, no doubt about that.

Although, with the EVP level fromdata functions, things may have shifted a bit? We should confirm among ourselves that this is the right move, though...

Reply:
We did decide to not deprecate thees.  They are still in `ec.h` but over zealous copy paste resulted in the pod file being modified.  I've reverted.